daos-cortx/docs/setup_robinhood_policy_engine.md

```diff
@@ -0,0 +1,149 @@
+
+* Edit the configuration file
+
+In 'General' block, set filesystem root path, and the corresponding file system type.
+       
+       fs_path = "/fs/root";
+       fs_type = xfs;
+ 
+* In 'ListManager' block, set database connection parameters:
+
+      db = rbh_fsname;
+      password_file = "/etc/robinhood.d/.dbpassword" ;
+
+It is recommended to define your fileclasses before running the initial file system scan.
+
+This way, you will get relevant information in 'rbh-report --class-info' report after the initial scan is completed.
+
+    fileclass empty_file {
+        definition { type == file and size == 0 }
+     }
+     fileclass small_file {
+        definition { type == file
+                 and size > 0
+                 and size <= 32MB }
+     }
+
+Reference config file is available [here.](https://github.com/Seagate/cortx-experiments/blob/rajkumarpatel2602-robinhood-pengine/daos-cortx/src/samples/posix.conf)
+Make sure to make above relevant changes in fields mentioned above.
+
+* Start Robinhood scan and update
+
+ `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/posix.conf`
+
+* Query database
+ 
+ `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/rbh-report --fs-info --class-info`
+ 
+# Test for daos to CORTX and vice-versa data movement
+
+In this exercise object movements from daos to cortx and cortx to daos will be made sure. This exercise will be carried out by defining policies inside a container and running those policies using robinhood policy engine. There are two prerequisites to carry out this test mentioned below.
+
+* Daos server node running on one VM
+
+  - follow this [document](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/docs/setup_daos.md) to setup daos and creating container.
+  
+* Cortx server node running on second VM
+
+  - follow this [document](https://github.com/Seagate/cortx/blob/main/QUICK_START.md) to setup cortx on your vm.
+  
+* Once everything is in-place let's start with the data movement test by following the steps below on daos node which is hosting robinhood policy engine.
+ 
+1. Create objects in a daos container (i.e. inside dfuse mount point)
+
+- create larger and smaller objects inside the container. container is mounted at /mnt/daos_container for this exercise and same fs_path will be used in config file as well.
+
+- These object(s) are going to be moved to s3 bucket and downloaded from there to doas container directory.
+
+2. Create config file and add policies and start Scanning the database and check contents of the container on robinhood's database
+
+- Readily available config files with correct config options are present [here.](https://github.com/Seagate/cortx-experiments/blob/rajkumarpatel2602-robinhood-pengine/daos-cortx/src/samples/rh_daos_cortx.conf)
```

Comment:
daos-cortx/src/samples/rh_daos_cortx.conf

Reply:
Hyperlink to this file used PR name in itself as this file we are looking for is introduced in this PR only. I will fix and in future use link which will be used on main.

Reply:
done